spacelift/internal/structs/user_group.go

```diff
@@ -0,0 +1,12 @@
+package structs
+
+type SpaceAccessRule struct {
+	Space            string `graphql:"space"`
+	SpaceAccessLevel string `graphql:"spaceAccessLevel"`
+}
+
+type UserGroup struct {
```

Comment:
There should be a field that indicates the IdP for this user group.

Reply:
The IdP is not controlled by the user, and it would be a computed fueld.

I think it will only make more sense to add more of these fields if we make this a data source. In the 1st version of the resource I'd suggest focusing on things that users need to manage access.

Reply:
That's fine to me.